app/controllers/incomeSources/add/IncomeSourcesAccountingMethodController.scala

```diff
@@ -104,41 +108,56 @@ class IncomeSourcesAccountingMethodController @Inject()(val authenticate: Authen
     )(user, messages)))
   }
 
+  def convertToIncomeSourceType(incomeSource: String): Either[Exception, IncomeSourceType] = {
```

Comment:
* there is already such conversion specified in /Users/hmrc/devcore/itvc/income-tax-view-change-frontend/app/enums/IncomeSourceJourney/IncomeSourceType.scala
if we can use single method instead of duplicating;

Reply:
Ok, replaced. This simplified a lot